### ISSUE TITLE
TCAMT won't save

### ISSUE THREAD

**Issue by RikiMerrick** [Jungyubw/newTCAMT#187]
I made a few changes to test story Test Step: LRI_4.0_1.1-GU;
for example:
description should have text:
Mr. William A. Jones is a 51 year old white male who presented with a complaint of diarrhea times 3 days. Dr. Nicholas Radon ordered a stool culture test to be performed. A stool specimen for the microbiology test was collected from the patient and sent to the clinical lab at Century Hospital, 2070 Test Park, Los Angeles, CA, 90067, for processing. The stool specimen was analyzed and preliminary result are sent 24 hours later, identifying E. coli O157:H7, Salmonella and Shigella flexneri.

I hit save and waited 15 min, it still said unsaved changes, I logged out and when I logged back in it showed last date modified as 2017/05/04 10:45:09.

**Comment by Jungyubw** [comment]
Hello Riki,
Sorry for inconvenience. 
Are you working "Eric" account? If so, I have checked but it is working correctly at my side. On Monday, I will check more logs at the office.

Thanks.

**Comment by RikiMerrick** [comment]
Yes I am working on Eric's profile - thanks for checking...weird that it
works for you.

Riki

Riki Merrick
Vernetzt, LLC
rikimerrick@gmail.com
cell: 916.216.0052
skype: riki.merrick
